.github/workflows/release-candidate.yaml

```diff
@@ -7,22 +7,14 @@ jobs:
   rc:
     name: Push release candidate tag
     runs-on: ubuntu-latest
-    container: public.ecr.aws/eksctl/eksctl-build:26e3c36557da8ae2db5d995cea673e05cc60cfce
```

Comment:
removing the build image was previously reverted because these `container` references were not cleaned up from the CI scripts